extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -135,9 +135,6 @@ export class SafeframeHostApi {
     /** @private {?Element} */
     this.iframe_ = null;
 
-    /** @private {?IntersectionObserver} */
-    this.intersectionObserver_ = null;
```

Comment:
(Note: this is just a minor cleanup. Does not affect the bug, just not needed anymore)